src/lib/FreeC/Pass/TypeInferencePass.hs

```diff
@@ -452,7 +431,6 @@ inferFuncDeclTypes' funcDecls = withLocalState $ do
     -- This is required such that polymorphically recursive functions don't
     -- cause a type error.
```

Comment:
```suggestion
  -- This is required such that polymorphically recursive functions don't
  -- cause a type error.
```
The comments are also misaligned on the two lines above, but i can't comment on them for some reason.

Reply:
Fixed by 7208d091314bd736fa3fed8a2f2bc7aac095085e.